components/Nav/NavListItem/NavListItem.js

```diff
@@ -50,9 +68,9 @@ const NavListItem = ({ sublinks, href, shouldPrefetch, name }) => {
             aria-haspopup={hasSublinks}
             aria-label="submenu"
             className={styles.sublinkToggleButton}
-            onClick={() => changeSublinkVisibility(!areSublinksVisible)}
-            onMouseEnter={() => changeSublinkVisibility(true)}
-            onMouseLeave={() => changeSublinkVisibility(false)}
+            onClick={() => setSublinksVisible(!areSublinksVisible)}
```

Comment:
Doing `setState(!stateVariable)` can lead to race conditions.

You'll want to do: `setState(prevState => !prevState)`

Reply:
my bad, missed it